src/qt/bitcoingui.cpp

```diff
@@ -627,7 +627,9 @@ void BitcoinGUI::createTrayIconMenu()
     trayIconMenu->addAction(verifyMessageAction);
     trayIconMenu->addSeparator();
     trayIconMenu->addAction(optionsAction);
-    trayIconMenu->addAction(openRPCConsoleAction);
+    if (walletFrame) {
```

Comment:
Why not `if (enableWallet) {`?

Reply:
I would prefer `if (enableWallet)` as well. Using `walletFrame` seems to be an older convention: https://github.com/bitcoin/bitcoin/commit/b7f4b6d35da7b63529a728eb9333fe424b2a5c6d